bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -384,4 +385,19 @@ public static Event createEvent(@NonNull Throwable exc,
     public static Logger getLogger() {
         return getClient().getConfig().getLogger();
     }
+
+    /**
+     * Retrieves an instantiated plugin of the given type, or null if none has been created
+     */
+    @SuppressWarnings("unchecked")
+    @Nullable
+    public static <T extends Plugin> T getPlugin(@NonNull Class<T> clz) {
```

Comment:
I think we should consider making a getPlugin method in Client and simply using the interface class to wrap it. This avoids anything else need to know about the PluginClient and having to open up the the `pluginClient` and `plugins` member variables for access here.

Reply:
I'm happy to go with the approach of adding a `getPlugin()` method to `Client`. Although I'm a bit wary in a general sense that there's a danger of us adding _every_ method like this onto `Client`, which is already quite a large class.

Reply:
Agreed on the general point, although I feel like in this case it's reasonable.